### ISSUE TITLE
Screen reader does not announce state as collapsed on submenu item control when focus moves on it.

### ISSUE THREAD

**Issue by RajeshGoriga** [microsoft/fluentui#19098]
### Environment Information

- **Package version(s)**: 8.22.0
- **Browser and OS versions**:  Chrome  Version 91.0.4472.164 ,NVDA 2020

### Describe the issue:

When using a screen reader, does not announce 

### Please provide a reproduction of the issue in a codepen:

 This is present in the Fluent samples here: 
[Sample example](https://fabricweb.z5.web.core.windows.net/oufr/8.22.1/#/examples/commandbar)

#### Actual behavior:
Announced like "New subMenu 1 of 6"

#### Expected behavior:

Announced like "New subMenu  collapsed 1 of 6"

### Documentation describing expected behavior

[MAS 4.2.1 – Object Information](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/doc2.aspx?sourcedoc=%7BEEE97BBD-9990-48A0-8B03-6EA775581AA8%7D&file=MAS%204.2.1%20%E2%80%93%20Object%20Information.docx&wdLOR=cBC823FF2-F30B-4029-A49F-90FCC0AB9FEB&action=default&mobileredirect=true&cid=2807a07d-ebc7-4de4-aafc-93223bc52ed5)

**Comment by smhigley** [comment]
NVDA doesn't announce expanded/collapsed when landing on a collapsed menuitem; this is by design on NVDA's part. You can check the behavior against the ARIA Practices menubar example: https://w3c.github.io/aria-practices/examples/menubar/menubar-editor.html, where NVDA also doesn't announce "collapsed" for any of the menu items with submenus.
